cmd/cayley/cayley.go

```diff
@@ -49,6 +49,8 @@ var (
 	configFile         = flag.String("config", "", "Path to an explicit configuration file.")
 	databasePath       = flag.String("dbpath", "/tmp/testdb", "Path to the database.")
 	databaseBackend    = flag.String("db", "memstore", "Database Backend.")
+	dumpFile           = flag.String("dump", "dbdump.nq", `Quad file to dump the database to (".gz" supported, "-" for stdout).`)
+	dumpType           = flag.String("dump_type", "json", `Quad file format ("json", "nquad").`)
```

Comment:
I would call this "quad" rather than nquad since we don't validate that the output is legal nquad.